### ISSUE TITLE
[BUG] KeyError while obfuscating a package with RFT mode

### ISSUE THREAD

**Issue by AlecHaring** [dashingsoft/pyarmor#1054]
I encountered an error while trying to obfuscate a Python package using Pyarmor 8.0.22's RFT mode

**Executed command**:
```
pyarmor gen --recursive --enable-rft .\pyarmor_test
```

**Output**:
```
INFO     Python 3.9.12
INFO     Pyarmor 8.0.22 (pro), ...
INFO     search inputs ...
INFO     find package at pyarmor_test
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     target platforms {'windows.amd64'}
INFO     write dist\pyarmor_runtime_00....\pyarmor_runtime.pyd
INFO     patch runtime file
INFO     generate runtime files OK
INFO     start to obfuscate scripts
INFO     process resource "pyarmor_test"
INFO     obfuscating file exception.py
CRITICAL unknown error, please check pyarmor.error.log
ERROR    'pyarmor_test.exception'
```

**pyarmor.error.log**:
```
11176 MainProcess 2023-03-30 13:54:10,804
Traceback (most recent call last):
  ...
  File "<maker>", line 714, in generate_obfuscated_script
  File "<maker>", line 504, in process
  File "<maker>", line 504, in <listcomp>
  File "<rftmaker>", line 17, in process
  File "<rftmaker>", line 329, in process
  File "<rftmaker>", line 305, in _refactor_tree
  File "<rftmaker>", line 297, in visit
  File "<rftmaker>", line 297, in visit
  File "<rftmaker>", line 297, in visit
  [Previous line repeated 3 more times]
  File "<rftmaker>", line 289, in visit
  File "<rftmaker>", line 182, in _rename_attrnode
KeyError: 'pyarmor_test.exception'
```

File structure:
```
pyarmor-test/
│
└── pyarmor_test/
    ├── exception.py
    └── main.py
```

[pyarmor-test.zip](https://github.com/dashingsoft/pyarmor/files/11114689/pyarmor-test.zip)

- I have tried with and without an `__init__.py` file in `pyarmor_test/`.
- Without `--enable-rft`, it obfuscates without a problem

I have also tried:
```
pyarmor gen --recursive --enable-rft .\pyarmor_test\main.py
```
But it seems like `--recursive` is ignored, and it only obfuscates `main.py`

**Comment by jondy** [comment]
Got it, I'll check it.

`--recursive` is used for a path, `gen` is designed to obfuscate packages or scripts list in the command line.

**Comment by jondy** [comment]
It's same issue as #1051 

I have reproduced it, and will be fixed in next release.

**Comment by AlecHaring** [comment]
Huh, weird. I'm still running into the same error. Same files, same command, but with 8.0.26

**Comment by jondy** [comment]
Reproduced, it's not same issue as #1051

**Comment by jondy** [comment]
Fixed in dev version `8.1.dev1`

**Comment by AlecHaring** [comment]
Confirmed, fixed
